### DISCUSSION TITLE
Installation an Batterie-Wechselrichter

### DISCUSSION THREAD

**Discussion by Flippo2906** [camueller/SmartApplianceEnabler#397] [Q&A]
Guten Tag zusammen,
ich habe da mal eine Frage. Ich habe einen Victron Multiplus II GX Batterie-Wechselrichter (1-Phasig). Dieser hat zwar die Internet-Schnittstelle, ist aber inkompatibel zu SMA HM2.0.
Gibt es eine Möglichkeit Stromeinspeisung und Stromverbrauch des Wechselrichters zu messen und dann im Portal anzeigen zu lassen?  Klaro ist mir bekannt, das man den Stromverbrauch eines Geräts messen kann, kann aber auch das einspeisen erkannt werden und an das Portal übermittelt werden als "Batterie"?
(Edit: ist mir grade eingefallen: Kann man vielleicht den Wechselrichter direkt auslesen ober Modbus???)
Welchen Strommesser würde ich dann benötigen?
Vielen Dank und Grüße
Philipp

**Comment by camueller** [comment]
Das SEMP-Protokoll, dass der SAE implementiert, ist dafür nicht geeignet.
